docs/tutorials/adding-new-project.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Adding a new project
+subject: Projects
```

Comment:
The subject is used to group pages into categories. (the word "categories" is already used for a special purpose by jekyll).

Since most of the categories are "things you can do with your EV3", I'm thinking we should make a new subject for "things you can do with the www.ev3dev.org webiste". I think a subject of "Projects" is too narrow and will only have this one thing listed under it.

So, `subject: "www.ev3dev.org Website"` or `subject: "This Website"`. What do you think?